variants/classical/orders/build.go

```diff
@@ -4,13 +4,11 @@ import (
 	"fmt"
 	"time"
 
-	cla "github.com/zond/godip/variants/classical/common"
 	dip "github.com/zond/godip/common"
+	cla "github.com/zond/godip/variants/classical/common"
 )
 
-func init() {
-	generators = append(generators, func() dip.Order { return &build{} })
-}
+var BuildGenerator func() dip.Order = func() dip.Order { return &build{} }
```

Comment:
From zond: "See previous comment on generators.

Etc. in the rest of the PR :)"